rollouts/docs/running-locally.md

```diff
@@ -0,0 +1,176 @@
+# Running Rollouts Locally
+
+## Prerequisites
+
+To run Rollouts locally, you will need:
+
+* Linux machine (technically it is possible to run Rollouts locally on a Mac but
+  due to differences in Docker between Linux and Mac, the Rollouts scripts are
+  confirmed to work on Linux)
+* [go 1.19](https://go.dev/dl/) or newer
+* `make`
+* Access to GKE clusters.
+
+## Cluster Setup
+
+### Creating a management cluster with KCC running
+First, you must create a config-controller cluster. You can follow the instructions on the 
+[config-controller-setup guide](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup) to create a config-controller cluster. Make sure your kubeconfig is 
+connected to this cluster.
+
+### Provisioning child clusters
+The next step will be to provision new child clusters. This example names the child cluster `gke-n`;
+you can replace `gke-n` with whatever name you want for your child cluster. 
+
+```sh
+# first step is to create a deployable instance of the package
+kpt pkg get git@github.com:droot/kpt-packages.git/cluster@main gke-n --for-deployment
```

Comment:
Should we move these packages to `kpt-samples`?

Reply:
Actually, I am adding support for [`kind` clusters](https://github.com/GoogleContainerTools/kpt/pull/3875), we will recommend that for running locally. So this package will probably be temporary.

Reply:
Awesome, I'll update these docs after the kind support is merged.